src/js/models/v2/aboutTheData/CoreReportingRow.js

```diff
@@ -8,25 +8,25 @@ import { format } from 'date-fns';
 
 const CoreReportingRow = {
     populateCore(data) {
-        this._budgetAuthority = data.current_total_budget_authority_amount || 0;
+        this._budgetAuthority = data.current_total_budget_authority_amount;
```

Comment:
Since the majority of these are listed as required on the backend and as required and a number. Is there a linked PR to update the api contract for the agencies/agency overview route properties that will now be nullable?

Reply:
Good catch